content/blog/changing-our-domian.md

```diff
@@ -12,10 +12,10 @@ feature_photo = ""
 feature_photo_alt = ""
 +++
 
-This morning, [Blain](/members/blain-smith/ "Blain Smith") educated me about the [history of the .io top-level domain](https://tamouse.github.io/blog/politics/2019/10/02/why-is-the-io-domain-problematic.html "Why is the .io domain problematic?"). I didn't know that history when I bought limeleaf.io last year before teaming up with Blain and Erik. 
+This morning, [Blain](/members/blain-smith/ "Blain Smith") educated me about the [history of the .io top-level domain](https://tamouse.github.io/blog/politics/2019/10/02/why-is-the-io-domain-problematic.html "Why is the .io domain problematic?"). I didn't know that history when I bought limeleaf.net last year before teaming up with Blain and Erik. 
 
 limeleaf.net wasn't available then, but to my surprise and delight, it was this morning, so I bought it. 
 
-All Limeleaf site traffic and email now redirect from limeleaf.io to limeleaf.net, and I have donated $50 to [Chagossian Voices](https://chagossianvoices.org/).
+All Limeleaf site traffic and email now redirect from limeleaf.net to limeleaf.net, and I have donated $50 to [Chagossian Voices](https://chagossianvoices.org/).
```

Comment:
This isn't a change that should occur ;-)

Reply:
Good catch!